wgsl/index.bs

```diff
@@ -12062,6 +12062,9 @@ Note: WGSL does not have zero built-in functions for [=atomic types=],
       <td>Construct [=zero value=] of the type `T`.
 </table>
 
+Note: Zero-filled vectors of AbstractInt can be written as
```

Comment:
Is there a subtle difference between zero-filled and zero-valued or does this PR just introduce a synonym? Thank you!

Reply:
SGTM, thanks!